flake.nix

```diff
@@ -2,14 +2,16 @@
   description = "A flake for the KEVM Semantics";
 
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/b01f185e4866de7c5b5a82f833ca9ea3c3f72fc4";
+    nixpkgs.url =
+      "github:NixOS/nixpkgs/b01f185e4866de7c5b5a82f833ca9ea3c3f72fc4";
     k-framework.url = "github:runtimeverification/k/v5.6.110";
     k-framework.inputs.nixpkgs.follows = "nixpkgs";
     #nixpkgs.follows = "k-framework/nixpkgs";
     flake-utils.follows = "k-framework/flake-utils";
     rv-utils.url = "github:runtimeverification/rv-nix-tools";
     poetry2nix.follows = "pyk/poetry2nix";
-    blockchain-k-plugin.url = "github:runtimeverification/blockchain-k-plugin/23fb5e8fdb4a87451fd40a0e49118eaab481931b";
+    blockchain-k-plugin.url =
+      "github:runtimeverification/blockchain-k-plugin/23fb5e8fdb4a87451fd40a0e49118eaab481931b";
```

Comment:
This must stay formatted how it was before, or `.github/workflows/update-version.yml` must be updated to recognize the new format here: https://github.com/runtimeverification/evm-semantics/blob/cb789ee9d856216e9b4dbc3fe5bbe3e775ce61fd/.github/workflows/update-version.yml#L69

Reply:
oh my bad, i ran nixfmt formatter out of habit